unattended_scripts/unattended_installation.sh

```diff
@@ -56,6 +56,14 @@ main() {
         exit 1;
     fi   
 
+    MACHINE_TYPE=`uname -m`
+
+    if ! { [ ${MACHINE_TYPE} = 'x86_64' ] || [ ${MACHINE_TYPE} = 'aarch64' ]; }; then
+    #if [ ${MACHINE_TYPE} == 'x86_64' ]; then
+        echo "Error: Package installation requires x86_64 or aarch64 architecture."
+        exit 1;
+    fi   
+
```

Comment:
Not necessary, the check already exists

https://github.com/wazuh/wazuh-packages/blob/5eda12e6ecb5eb4538830ead1c6163195a025af1/unattended_scripts/install_functions/opendistro/common.sh#L25

```
checkArch() {
    arch=$(uname -m)

    if [ ${arch} != "x86_64" ]; then
        logger -e "Uncompatible system. This script must be run on a 64-bit system."
        exit 1;
    fi
}
```